src/api/controllers/risk.js

```diff
@@ -0,0 +1,43 @@
+const fetch = require('node-fetch');
+
+export const YIELD_RISK_API_EXPONENTIAL =
+  'https://api.exponential.fi/api/pool-risk/search';
+
+const getRiskRating = async (req, res) => {
+  const poolData = req.body;
```

Comment:
lets add a try catch in case their api is down etc. 

in catch i'd add something like; 
```
res.status(503).json({
  status: 'error',
  message: 'Exponential API unavailable',
});
```